documentation/disaster-recovery.md

```diff
@@ -103,25 +107,25 @@ In case the database instance is lost, the objectives are:
 
 ### Recreate the lost postgres database instance
 
-In case the database service is deleted or in an inconsistent state we must recreate it and repopulate it.
+Please note, this process could take about circa 60 min to compleate. In case the database service is deleted or in an inconsistent state we must recreate it and repopulate it.
 First make sure it is fully gone by running cf services and cf delete-service if necessary.
-Then recreate the lost postgres database instance using the makefile recipes `terraform-app-plan` and `terraform-app-apply`:
+Then recreate the lost postgress database instance using the following make recipes `terraform-app-plan` and `terraform-app-apply`:
 
 ```
-make <env> terraform-app-plan passcode=MyPasscode tag=main-08406f04dd9eadb7df6fcda5213be880d7df37ed-20201022090714 ## to see the proposed changes.
+[ aws-vault exec Deployments -- ] make <env> terraform-app-plan passcode=MyPasscode tag=main-08406f04dd9eadb7df6fcda5213be880d7df37ed-20201022090714 ## to see the proposed changes.
 ```
 
 ```
-make <env> terraform-app-apply passcode=MyPasscode tag=main-08406f04dd9eadb7df6fcda5213be880d7df37ed-20201022090714 ## apply proposed changes i.e. create new database instance.
+[ aws-vault exec Deployments -- ] make <env> terraform-app-apply passcode=MyPasscode CONFIRM_RESTORE=Yes tag=main-08406f04dd9eadb7df6fcda5213be880d7df37ed-20201022090714 ## apply proposed changes i.e. create new database instance.
 ```
 This will create a new postgres database instance as described in the terraform configuration file.
 
 ### Restore Data From Daily Backup
 
-Once the lost database instance has been recreated, the last daily backup will need to be restored. To achieve this, use the makefile recipe `restore-daily-backup`. The following would need to be set: `passcode` (a [GOV.UK PaaS one-time passcode](https://login.london.cloud.service.gov.uk/passcode)), `CONFIRM_PRODUCTION` (true) and `tag` (cf app teaching-vacancies-production | grep "docker image" the tag is after the : i.e ghcr.io/dfe-digital/teaching-vacancies:main-7b736906654cbd42145420ad40fcbc6ec257bd1c)
+Once the lost database instance has been recreated, the last daily backup will need to be restored. To achieve this, use the following makefile recipe `restore-data-from-backup`. The following would need to be set `passcode` (a [GOV.UK PaaS one-time passcode](https://login.london.cloud.service.gov.uk/passcode)), `CONFIRM_PRODUCTION` (true) and `tag` (cf app teaching-vacancies-production | grep "docker image" the tag is after the : i.e ghcr.io/dfe-digital/teaching-vacancies:main-7b736906654cbd42145420ad40fcbc6ec257bd1c)
```

Comment:
use the following makefile recipe: `restore-data-from-backup`

But doesn't matter that much

Reply:
The following would need to be set: `passcode` (a [GOV.UK PaaS one-time passcode](https://login.london.cloud.service.gov.uk/passcode)) ...